extensions/mentions/js/src/forum/utils/textFormatter.js

```diff
@@ -31,3 +31,17 @@ export function filterPostMentions(tag) {
     return true;
   }
 }
+
+export function filterGroupMentions(tag) {
+  const group = app.store.getById('groups', tag.getAttribute('id'));
+
+  if (group) {
+    tag.setAttribute('displayname', extractText(group.namePlural()));
```

Comment:
just a terminology nitpick, can we change all `diplayname` occurrences to groupname or just `name`? `displayname` is such a user-specific term in the codebase right now.